ci/release/_build.sh

```diff
@@ -4,19 +4,23 @@ cd -- "$(dirname "$0")/../.."
 . ./ci/sub/lib.sh
 
 sh_c mkdir -p "$HW_BUILD_DIR"
-sh_c rsync --recursive --perms --delete \
-  --human-readable --copy-links ./ci/release/template/ "$HW_BUILD_DIR/"
-VERSION=$VERSION sh_c eval "'$HW_BUILD_DIR/README.md.sh'" \> "'$HW_BUILD_DIR/README.md'"
-sh_c rm -f "$HW_BUILD_DIR/README.md.sh"
-sh_c find "$HW_BUILD_DIR" -exec touch {} \\\;
+sh_c cp ./ci/release/template/LICENSE.txt "$HW_BUILD_DIR"
+sh_c cp ./ci/release/template/Makefile "$HW_BUILD_DIR"
+sh_c cp -R ./ci/release/template/man "$HW_BUILD_DIR"
+sh_c cp -Rf ./ci/release/template/scripts "$HW_BUILD_DIR"
+sh_c VERSION="$VERSION" ./ci/release/template/README.md.sh \> "'$HW_BUILD_DIR/README.md'"
 
 ensure_goos
 ensure_goarch
 sh_c mkdir -p "$HW_BUILD_DIR/bin"
-sh_c CGO_ENABLED=0 go build -trimpath \
+sh_c GOOS="$GOOS" GOARCH="$GOARCH" CGO_ENABLED=0 go build -trimpath \
```

Comment:
we don't need the cgo_enabled=0 anymore, unless i'm missing something

Reply:
Go defaults to using libc for network resolution and os/user both of which I believe we import. `CGO_ENABLED=0` turns the libc integration off and uses pure Go implementations which should work in 99% of use cases and enable our binaries to be significantly more portable.